src/style.css

```diff
@@ -1 +1,76 @@
 /* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 1200px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+  margin: 0 auto;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+@media (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+}
+
+.header__logo {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+}
+
+.menu {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  font-family: "Roboto", sans-serif;
+}
+
+.menu__item {
+  list-style: none;
+  padding: 23px 0;
+}
+
+.menu__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.menu__link {
+  text-decoration: none;
+  padding: 23px 0;
+  cursor: pointer;
+  color: #000;
+}
+
+.menu__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00acdc;
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
```

Comment:
🔥🔥🔥
Also, you can add 0 for left and this will the same but without transform

(Don't change your variant. Your code is cool too 😉)